test/lint/lint-locale-dependence.py

```diff
@@ -0,0 +1,259 @@
+    "isgraph",      # LC_CTYPE
+    "islower",      # LC_CTYPE
+    "isprint",      # LC_CTYPE
+    "ispunct",      # LC_CTYPE
+    "isspace",      # LC_CTYPE
+    "isupper",      # LC_CTYPE
+    "iswalnum",     # LC_CTYPE
+    "iswalpha",     # LC_CTYPE
+    "iswblank",     # LC_CTYPE
+    "iswcntrl",     # LC_CTYPE
+    "iswctype",     # LC_CTYPE
+    "iswdigit",     # LC_CTYPE
+    "iswgraph",     # LC_CTYPE
+    "iswlower",     # LC_CTYPE
+    "iswprint",     # LC_CTYPE
+    "iswpunct",     # LC_CTYPE
+    "iswspace",     # LC_CTYPE
+    "iswupper",     # LC_CTYPE
+    "iswxdigit",    # LC_CTYPE
+    "isxdigit",     # LC_CTYPE
+    "localeconv",   # LC_NUMERIC + LC_MONETARY
+    "mblen",        # LC_CTYPE
+    "mbrlen",
+    "mbrtowc",
+    "mbsinit",
+    "mbsnrtowcs",
+    "mbsrtowcs",
+    "mbstowcs",     # LC_CTYPE
+    "mbtowc",       # LC_CTYPE
+    "mktime",
+    "normalize",    # boost::locale::normalize
+    "printf",       # LC_NUMERIC
+    "putwc",
+    "putwchar",
+    "scanf",        # LC_NUMERIC
+    "setlocale",
+    "snprintf",
+    "sprintf",
+    "sscanf",
+    "std::locale::global",
+    "std::to_string",
+    "stod",
+    "stof",
+    "stoi",
+    "stol",
+    "stold",
+    "stoll",
+    "stoul",
+    "stoull",
+    "strcasecmp",
+    "strcasestr",
+    "strcoll",      # LC_COLLATE
+    #"strerror",
+    "strfmon",
+    "strftime",     # LC_TIME
+    "strncasecmp",
+    "strptime",
+    "strtod",       # LC_NUMERIC
+    "strtof",
+    "strtoimax",
+    "strtol",       # LC_NUMERIC
+    "strtold",
+    "strtoll",
+    "strtoq",
+    "strtoul",      # LC_NUMERIC
+    "strtoull",
+    "strtoumax",
+    "strtouq",
+    "strxfrm",      # LC_COLLATE
+    "swprintf",
+    "to_lower",     # boost::locale::to_lower
+    "to_title",     # boost::locale::to_title
+    "to_upper",     # boost::locale::to_upper
+    "tolower",      # LC_CTYPE
+    "toupper",      # LC_CTYPE
+    "towctrans",
+    "towlower",     # LC_CTYPE
+    "towupper",     # LC_CTYPE
+    "trim",         # boost::algorithm::trim
+    "trim_left",    # boost::algorithm::trim_left
+    "trim_right",   # boost::algorithm::trim_right
+    "ungetwc",
+    "vasprintf",
+    "vdprintf",
+    "versionsort",
+    "vfprintf",
+    "vfscanf",
+    "vfwprintf",
+    "vprintf",
+    "vscanf",
+    "vsnprintf",
+    "vsprintf",
+    "vsscanf",
+    "vswprintf",
+    "vwprintf",
+    "wcrtomb",
+    "wcscasecmp",
+    "wcscoll",      # LC_COLLATE
+    "wcsftime",     # LC_TIME
+    "wcsncasecmp",
+    "wcsnrtombs",
+    "wcsrtombs",
+    "wcstod",       # LC_NUMERIC
+    "wcstof",
+    "wcstoimax",
+    "wcstol",       # LC_NUMERIC
+    "wcstold",
+    "wcstoll",
+    "wcstombs",     # LC_CTYPE
+    "wcstoul",      # LC_NUMERIC
+    "wcstoull",
+    "wcstoumax",
+    "wcswidth",
+    "wcsxfrm",      # LC_COLLATE
+    "wctob",
+    "wctomb",       # LC_CTYPE
+    "wctrans",
+    "wctype",
+    "wcwidth",
+    "wprintf"
+]
+
+
+def git_grep_call(regexp_locale_dependent_functions):
```

Comment:
This function name is too generaic for what it does. Maybe `find_locale_dependent_function_uses`?